Tasks/InstallSSHKeyV0/installsshkey-util.ts

```diff
@@ -227,12 +236,56 @@ export function setKnownHosts(knownHostsEntry: string) {
         knownHostsContent = fs.readFileSync(knownHostsFile).toString();
     }
 
+    tl.debug('Inserting entry into known_hosts');
+    const taskAlreadyUsed = tl.getVariable(postKnownHostsLocationSetting);
+    if (taskAlreadyUsed) {
+        fs.appendFileSync(knownHostsFile, knownHostsEntry + os.EOL);
+    } else {
+        fs.writeFileSync(knownHostsFile, knownHostsEntry + os.EOL);
+    }
+
     tl.setTaskVariable(postKnownHostsContentsSetting, knownHostsContent);
-    tl.setTaskVariable(postKnownHostsLocationSetting, knownHostsFile);
+    tl.setVariable(postKnownHostsLocationSetting, knownHostsFile);
     tl.setTaskVariable(postKnownHostsDeleteFileSetting, knownHostsDeleteFileOnClose);
+}
 
-    tl.debug('Inserting entry into known_hosts');
-    fs.writeFileSync(knownHostsFile, knownHostsEntry + os.EOL);
+export function addConfigEntry(configEntry: ConfigFileEntry): void {
+    const configFolder: string = path.join(os.homedir(), '.ssh');
+    const configFilePath: string = path.join(configFolder, 'config');
+    let configFileContent: string = '';
+    let deleteConfigFileOnClose: string = 'true';
+    if (!fs.existsSync(configFolder)) {
+        fs.mkdirSync(configFolder);
+    } else if (fs.existsSync(configFilePath)) {
+        tl.debug('Reading config file');
+        deleteConfigFileOnClose = '';
+        configFileContent = fs.readFileSync(configFilePath).toString();
+    }
+
+    const configEntryContent: string = configEntry.toString();
+    console.log(tl.loc("InsertingIntoConfig"));
+    console.log(configEntryContent);
+    const configAlreadyChanged = tl.getTaskVariable(postConfigLocationSetting);
+    if (configAlreadyChanged) {
+        fs.appendFileSync(configFilePath, os.EOL + configEntryContent);
```

Comment:
Nit: could you use js templates? Just to be consistent - since you used it further?